.github/workflows/ci_tests.yml

```diff
@@ -201,6 +201,8 @@ jobs:
       ENABLE_TRILINOS: OFF
       GCP_BUCKET: geosx/integratedTests
       RUNS_ON: ubuntu-22.04
+#      RUNS_ON: self-hosted
+#      DOCKER_RUN_ARGS: "--cpus=8 --memory=128g"
```

Comment:
Remove?

Reply:
this will be used in the follow on the `integratedTests` pr, but it isn't hard to find elsewhere. I can remove it.

Reply:
Yes, they are just below, lines 251-252.
That would let the commented parts in `build_and_test.yml` only.